### PR TITLE
Check for (some) packages needed for installation to complete.

### PR DESCRIPTION
**Pull request by ltratt**
This is certainly not a complete list, but I noticed earlier that if these things are missing, then installation fails in weird or silent fashion (e.g. it appears to continue successfully, but doesn't install things, and scrolling back through the terminal one eventually spots the source of the error). It would be good to think more carefully about what other things are needed, but this is a
start at least.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,5 +1,23 @@
 #! /bin/sh
 
+which git 2> /dev/null > /dev/null
```
Comment (vext01) [inline build.sh]: You could shorten this to `which git > /dev/null 2>&1` I think.

File: build.sh [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+if [ $? -ne 0 ]; then
+    echo "git must be installed" > /dev/stderr
```
Comment (vext01) [inline build.sh]: And these can be shortened to `>&2 echo "..."`

Comment (ltratt) [reply build.sh]: Shorter, maybe, but clearer ... ?

Comment (vext01) [reply build.sh]: Up to you. I don't feel strongly.

File: build.sh (hunk 1 above)
Comment (ltratt) [reply build.sh]: Agreed.

File: build.sh (hunk 2 above)
Comment (ltratt) [reply build.sh]: Although, on reflection, `echo ... >&2` is fine, given that we use similar notation earlier.

**Comment by ltratt** [comment]
OK, see if that new commit is what you were thinking of or not.

**Comment by vext01** [comment]
Looks good. Tested? Then we can squash and merge.

**Comment by ltratt** [comment]
Yep, tested and appears to work. Squashed.
